include/eve/detail/function/simd/arm/neon/swap_adjacent_groups.hpp

```diff
@@ -53,7 +53,12 @@ namespace eve::detail
         else  if constexpr( c == category::float64x2  )
         {
           if constexpr(current_api > asimd) return vextq_f64(v,v,1);
-          else                              return that_t(v.get(1),v.get(0));
+          else
+          {
+            auto u64 = vreinterpretq_u64_f64(v);
```

Comment:
does it gives better codegen than before ?

Reply:
At least it should. I mean it's one instruction.

Reply:
Oh...
So there is smth nuts going on:
https://godbolt.org/z/vqnr93fh6

We generate some ultra hot garbage. I have no idea what's up.

And apparently these casts should only be avaliable on aarch64,  but our gcc arm builds them fine. What's up with that?
https://godbolt.org/z/vd6M86Px3